packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2151,8 +2151,9 @@ export const audiusBackend = ({
         withTrendingTrack: true
       }
 
+      // Passing user_id to support manager mode
       const getNotificationsUrl = queryString.stringifyUrl({
-        url: `${identityServiceUrl}/notifications`,
+        url: `${identityServiceUrl}/notifications?user_id=${account.user_id}`,
```

Comment:
we shouldn't need to care about this too much. notifs haven't been on identity for over a year+ but this fallback was in place if you did scroll far enough.

Reply:
Yeah you're right. I thought this was how we were marking things and didn't notice until I got past the other errors that this didn't actually do anything. I have a separate ticket open to fix entity manager on DN, which we can do async from shipping.